package.json

```diff
@@ -7,7 +7,7 @@
     "build": "npx gulp build:dist",
     "build:prod": "npm run build -- --production",
     "deploy": "gh-pages -d docs/_site",
-    "docs:compile": "npm run docs:config && npm run build && npx gulp docs-site",
+    "docs:compile": "npm run docs:config && npm run build && npx gulp docs-build",
```

Comment:
we should be able to delete `npm run build` from here since `docs-build` calls `build:dist` same as in `npm run build`